builtin/logical/consul/path_roles.go

```diff
@@ -32,7 +32,19 @@ func pathRoles() *framework.Path {
 			"policy": &framework.FieldSchema{
 				Type: framework.TypeString,
 				Description: `Policy document, base64 encoded. Required
-for 'client' tokens.`,
+for 'client' tokens. Required for Consul pre-1.4`,
+			},
+
+			"policies": &framework.FieldSchema{
+				Type: framework.TypeCommaStringSlice,
+				Description: `List of policies attached to the token. Required
+for Consul 1.4 or above`,
+			},
+
+			"local": &framework.FieldSchema{
+				Type: framework.TypeBool,
+				Description: `Indicates that the token should not be replicated globally 
+and instead be local to the current datacenter.  Available in Consul 1.4 and above.`,
```

Comment:
curious what the default for this setting should be, what does consul default to?

Reply:
Consul defaults to false.